sdk/src/manifest_store.rs

```diff
@@ -146,12 +146,12 @@ impl ManifestStore {
     /// Loads a ManifestStore from a file
     pub async fn from_bytes_async(
         format: &str,
-        image_bytes: Vec<u8>,
+        image_bytes: &[u8],
```

Comment:
Changed title of PR to include (MINOR) since this changes an existing API. (Yes, I know, I asked for the change. :-) )